### ISSUE TITLE
Establishing connection to freeciv Server

### ISSUE THREAD

**Issue by yashbonde** [yashbonde/freeciv-python#3]
**Introduction:** Freeciv uses a server-client system to run the game where the information transfer is done using packets. The server can be run locally, on cloud, or using meta-server functionality of the game. Connecting to the server is key to running the package, once this part is complete and we can establish connection we would have completed 30% of the work.

**Problem:** To establish connection to the server we need to use set of packets. The packets used are defined [here](https://github.com/freeciv/freeciv/blob/master/common/networking/packets.def). It uses a non-standard packet structure and thus is difficult to use. We need to find a way to establish connection to the server using the proper packets.

**Approach:** you can define the properties of the server before running it as follows:
```
$ fcser --port=5000 --debug=4 --log=server_log.log
```

Here `port` refers to port so the server can be connected on `127.0.0.1:port`, `--debug=4` gives more detailed information and `--log=log_file` logs the information from the server (very helpful in understanding the structure)

**NOTES:** This issue also depends upon another issue #4 about converting packets to python.
